src/apis/auth.ts

```diff
@@ -0,0 +1,64 @@
+import { isAxiosError } from 'axios';
+
+import { ROUTES } from '../constants/routes';
+import { axiosInterface } from './axios';
+import { setLocalStorage, removeLocalStorage } from '../utils/storage';
+
+interface LoginParams {
+  email: string;
+  password: string;
+}
+
+interface SignupParams {
+  email: string;
+  fullName: string;
+  password: string;
+}
+
+// 회원가입
+export const signup = async (params: SignupParams) => {
+  await axiosInterface.post('signup', {
+    email: params.email,
+    fullName: params.fullName,
+    password: params.password,
+  });
+};
```

Comment:
```suggestion
export const signup = async ({ email, fullName, password }: SignupParams) => {
  await axiosInterface.post('signup', {
    email,
    fullName,
    password,
  });
};
```
이런 식으로 [구조 분해 할당(객체)](https://developer.mozilla.org/ko/docs/Web/JavaScript/Reference/Operators/Destructuring_assignment#%EA%B0%9D%EC%B2%B4_%EA%B5%AC%EC%A1%B0_%EB%B6%84%ED%95%B4)를 하면 좋을 것 같습니다!!
또한 의문인것은 `signup`에서 만약 email이 중복이면 에러를 api가 보내지 않나라는 생각을 해봅니다!!
`try catch`로 한 번 시도 해보시면 좋을 것 같습니다!!

Reply:
오 훨씬 보기 좋네요! 저도 작성시 참고 하겠습니다